app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryUiRenderer.kt

```diff
@@ -35,13 +35,21 @@ class NewMedicalHistoryUiRenderer(
           ui.renderAnswerForQuestion(DIAGNOSED_WITH_DIABETES, hasDiabetes)
         }
       }
+
+      renderHypertensionTreatmentQuestion(model)
     }
 
     ui.showDiagnosisRequiredError(model.showDiagnosisRequiredError)
 
     renderNextButton(model)
   }
 
+  private fun renderHypertensionTreatmentQuestion(model: NewMedicalHistoryModel) {
+    if (country.isoCountryCode == Country.INDIA && model.diagnosedWithHypertension) {
+      ui.showHypertensionTreatmentQuestion(model.ongoingMedicalHistoryEntry.isOnHypertensionTreatment)
+    }
```

Comment:
Deciding whether or not to show this question is business logic and must be handled in the update function which should dispatch an effect to show it.

I am approving this for now because this is urgent but we should correct this as soon as we have time.

Reply:
I thought about this approach, but it made more sense to show/hide the question view from UI renderer to make sure it gets restored properly after a config change or something like that. So the other option would be to pass the country when creating the model so that we can have a computed property to check-in view render. This check will still be present in the renderer, but the property will now be computed. wdyt?

Ideally, I want to avoid using effects for these sorts of things to avoid UI issues.

Reply:
That works as well. This should be verified as part of the model, not in the UI renderer.

Reply:
Yes, it will be present in the model. What I meant was we will use the computed property for the check in the UI renderer like so

```kt
if (model.showOngoingHypertensionTreatment) {
  // code
} else {
  // code
}
```

Sounds good?

Reply:
LGTM